pages/common/aws-dynamodb.md

```diff
@@ -0,0 +1,36 @@
+# aws dynamodb
+
+> CLI for AWS IAM.
```

Comment:
As far as I can remember, Dynamodb is a separate command and isn't part of IAM (which is managed by `aws iam`, so not sure about this description.